src/components/auth/landing.js

```diff
@@ -0,0 +1,321 @@
+                  class="img-fluid"
+                  alt=""
+                  srcset=""
+                />
+                <p>Work on projects/ Build a solid Portfolio</p>
+              </div>
+            </div>
+          </div>
+        </section>
+
+        <div class="description__section py-5">
+          <div class="container">
+            <div class="description__goals">
+              <img src="./img/goal_target.png" class="img-fluid" alt="" />
+              <div class="goals__texts">
+                <h3>Tap into the Power of Community Learning.</h3>
+                <p class="my-3">
+                  Community learning is a great way to learn, especially ones
+                  with dedicated mentors. Code Clan Code Clan Nigeria is the
+                  perfect place for you.
+                </p>
+                <a href="#">
+                  Learn More <i class="fas fa-arrow-right"></i>
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <section class="section-three bg-blue">
+          <h2 class="text-white container text-center">
+            WHY CHOOSE CODECLAN NIGERIA ?
+          </h2>
+
+          <div class="container">
+            <div>
+              <img src="img/svgs/undraw_community_8nwl 1.svg" alt="" />
+              <p class="text-white text-md">Tap into the power of community</p>
+            </div>
+
+            <div>
+              <img src="img/svgs/undraw_sharing_articles_t5aa 1.svg" alt="" />
+              <p class="text-white text-md">
+                Dedicated mentors assigned to clans
+              </p>
+            </div>
+
+            <div>
+              <img src="img/svgs/undraw_project_completed_w0oq 1.svg" alt="" />
+              <p class="text-white text-md">
+                Earn badges for courses completed
+              </p>
+            </div>
+
+            <div>
+              <img src="img/svgs/undraw_status_update_jjgk 1.svg" alt="" />
+              <p class="text-white text-md">Learn to work in a team</p>
+            </div>
+
+            <div>
+              <img src="img/svgs/undraw_quiz_nlyh 1.svg" alt="" />
+              <p class="text-white text-md">
+                Chance to become a mentor upon the completion of your program
+              </p>
+            </div>
+
+            <div>
+              <img src="img/svgs/undraw_about_me_wa29 1.svg" alt="" />
+              <p class="text-white text-md">
+                Chance to boost your portfolio with projects and challenges
+              </p>
+            </div>
+          </div>
+
+          <p class="text-white container text-center">
+            Resources and events to boost your skill in a matter of time
+          </p>
+        </section>
+        <section class="section-four container">
+          <div class="section-four-title">
+            <h2>Courses available</h2>
+            <a href="" class="text-black text-md">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5641179)